### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by eedorenko**

### CODE REVIEW

File: environment_setup/requirements.txt [hunk 1]
```diff
@@ -4,4 +4,5 @@ azureml>=0.2
 azureml-sdk>=1.0
 python-dotenv>=0.10.3
 flake8
-flake8_formatter_junit_xml
\ No newline at end of file
+flake8_formatter_junit_xml
+azure-cli==2.0.46
```
Comment (dtzar) [inline environment_setup/requirements.txt]: This is an extremely old version of `az` - could we do a quick test with the latest 2.0.71?

Comment (eedorenko) [reply environment_setup/requirements.txt]: Done. Works with 2.0.71

File: docs/code_description.md [hunk 2]
```diff
@@ -2,59 +2,37 @@
...
+- ml_service/pipelines/run_train_pipeline.py : invokes a published ML training pipeline via REST API.
+- ml_service/util : contains common utility functions used to build and publish an ML training pipeline.
 
-- 00-WorkSpace.py : This is a onetime run script. It reads the workspace details from ./aml_config/config.json file and create (if workspace not available) or get (existing workspace). 
+### Code
 
-- 01-Experiment.py : This is a onetime run script. It registers the root directory as project. It is not included as a step in build pipeline.
+- code/training/train.py : a training step of an ML training pipeline.
+- code/evaluate/evaluate_model.py : an evaluating step of an ML training pipeline.
+- code/evaluate/evaluate_model.py : registers a new trained model if evaluation shows the new model is more performent than the previous one.
```
Comment (dtzar) [inline docs/code_description.md]: Duplicate file name on two lines.

File: docs/getting_started.md [hunk 3]
```diff
@@ -10,211 +10,179 @@ We use Azure DevOps for running our build(CI), retraining trigger and release (C
...
-   - Release Deployment pipeline: QA environment is auto triggered when there is a new model.
-    Prod is manual only and user decides when to release to this environment.
+![workspace connection](./images/workspace-connection.png)
+
+Add an artifact to the pipeline and select **AzureML Model Artifact** source type. Select the **Service Endpoint** and **Model Names** from the drop down lists:
+
+![model artifact](./images/model-artifact.png)
+
+Create a stage **QA (ACI)** and add a single task to the job **Azure ML Model Deploy**. Specify task parameters as it is shown below:
+
```
Comment (dtzar) [inline docs/getting_started.md]: Please add the `code` snip for the file paths to make the user experience much easier

File: docs/getting_started.md [hunk 4]
```diff
@@ -10,211 +10,179 @@ We use Azure DevOps for running our build(CI), retraining trigger and release (C
...
+![deploy aci](./images/deploy-aci.png)
+
+In a similar way create a stage **Prod (AKS** and add a single task to the job **Azure ML Model Deploy**. Specify task parameters as it is shown below:
+
```
Comment (dtzar) [inline docs/getting_started.md]: Please add the `code` snip for the file paths to make the user experience much easier

File: docs/code_description.md (hunk 2 above)
Comment (eedorenko) [reply docs/code_description.md]: Done

File: docs/getting_started.md (hunk 4 above)
Comment (eedorenko) [reply docs/getting_started.md]: Done

File: docs/getting_started.md (hunk 3 above)
Comment (eedorenko) [reply docs/getting_started.md]: Done

**Pull request merged**